### PR TITLE
docs: add Massoud Maboudi's portfolio to showcase

### PR DESCRIPTION
**Pull request by massoudmaboudi**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation

Another RTL website from @massoudmaboudi 😁

### Test links

Deploy preview: https://deploy-preview-_____--docusaurus-2.netlify.app/

## Related issues/PRs

#7406

### CODE REVIEW

**Comment by massoudmaboudi** [comment]
any update on this PR?

**Comment by Josh-Cena** [comment]
@massoudmaboudi Seems you didn't do a clean merge?

Also, I'd like to get @slorber's confirmation that we'd like this in favorites (I'd like it to), but he's currently on holiday.

**Comment by massoudmaboudi** [comment]
> @massoudmaboudi Seems you didn't do a clean merge?
> 
> Also, I'd like to get @slorber's confirmation that we'd like this in favorites (I'd like it to), but he's currently on holiday.

@Josh-Cena 
I did, but things keep changing so I have to update it to resolve the conflicts.
Thanks

**Comment by Josh-Cena** [comment]
> I did, but things keep changing so I have to update it to resolve the conflicts.

Your branch is not merged—look at the diff, you still have a diverging branch.

**Comment by Josh-Cena** [comment]
OK, I've done a clean merge for you.

**Comment by slorber** [comment]
Thanks 👍 

I'm not really willing to add this one to favorites.

Your 2 RTL sites are not bad, but are not really as good as other favorite sites in terms of design and customization.
The reason your other site is in favorite is that it's the only RTL site we have in our showcase currently.

BTW your RTL favorite site is likely to be replaced one day by another better/larger site that also uses RTL.

Don't take this personally and it doesn't mean your sites are bad: we'd like to keep favorite sites count low (otherwise it doesn't mean anything anymore), maybe like 15-20 sites max, and some sites may lose their favorite label over time


If we have to keep only 1 RTL site in favorites, I'd rather keep your other site as at least it has some docs and more content, unlike this new one

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -886,6 +886,14 @@ const Users: User[] = [
     source: 'https://github.com/ggicci/httpin/tree/documentation/docs',
     tags: ['opensource'],
   },
+  {
+    title: 'I am Massoud',
+    description: 'The portfolio of Massoud Maboudi, Full Stack Developer',
+    preview: require('./showcase/iammassoud.png'),
+    website: 'https://iammassoud.ir/',
+    source: null,
+    tags: ['favorite', 'personal', 'rtl'],
```
Comment (slorber) [inline website/src/data/users.tsx:895]:
```suggestion
    tags: ['personal', 'rtl'],
```

**Comment by massoudmaboudi** [comment]
> Thanks 👍
> 
> I'm not really willing to add this one to favorites.
> 
> Your 2 RTL sites are not bad, but are not really as good as other favorite sites in terms of design and customization. The reason your other site is in favorite is that it's the only RTL site we have in our showcase currently.
> 
> BTW your RTL favorite site is likely to be replaced one day by another better/larger site that also uses RTL.
> 
> Don't take this personally and it doesn't mean your sites are bad: we'd like to keep favorite sites count low (otherwise it doesn't mean anything anymore), maybe like 15-20 sites max, and some sites may lose their favorite label over time
> 
> If we have to keep only 1 RTL site in favorites, I'd rather keep your other site as at least it has some docs and more content, unlike this new one

It's nothing about personal.

If there are other RTL websites then I would love to learn from them also so please don't hesitate to replace them.
But today I've changed my website theme and design completely. So it has nested pages and documents.

image
image
image

**Pull request merged**